### PR TITLE
keep old esm directory structure

### PR DESCRIPTION
**Pull request by promer94**
#932 accidentally changed the esm directory structure.

### CODE REVIEW

**Comment by promer94** [comment]
#932 accidentally changed the esm directory structure.

File: tsconfig.json [hunk 1]
```diff
@@ -24,7 +25,7 @@
   },
   "include": [
     "src/**/*",
```
Comment (shuding) [inline tsconfig.json:27]: Should we also update this?

Comment (promer94) [reply tsconfig.json:27]: #TIL 
> Importantly, rootDir does not affect which files become part of the compilation. It has no interaction with the include, exclude, or files tsconfig.json settings.
https://www.typescriptlang.org/tsconfig#rootDir

Comment (shuding) [reply tsconfig.json:27]: That is confusing, so `includes` only determines which definitions get built?

Comment (promer94) [reply tsconfig.json]: rootDir is used to keep the output's directory structure simple and clean 
includes determines which file (must be in rootDir) will be compiled by tsc

We should have a separate tsconfig.test.json which is only used for ts-jest
In this way ```jest-setup.ts``` will not affect the output's directory structure anymore

**Review by shuding — APPROVED** [review]
Thanks! ❤️

**Pull request merged**